save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/benchmarks/BenchmarkEntity.kt

```diff
@@ -19,6 +21,16 @@ enum class BenchmarkCategoryEnum {
     PERFORMANCE,
     STATIC_ANALYSIS,
     ;
+
+    companion object : TabMenuBar<BenchmarkCategoryEnum> {
+        // The string is the postfix of a [regexForUrlClassification] for parsing the url
+        private val postfixInRegex = values().map { it.name.lowercase() }.joinToString { "|" }
```

Comment:
Are you sure you want to get `"|, |, |, |"`?

Reply:
Yes, it is convenient for regex

Reply:
How can "|, |, |, |" be convenient for regex?

Reply:
Redone